packages/trpc/server/routers/viewer/attributes/create.schema.ts

```diff
@@ -4,7 +4,7 @@ export const createAttributeSchema = z.object({
   name: z.string(),
   type: z.enum(["TEXT", "NUMBER", "SINGLE_SELECT", "MULTI_SELECT"]),
   isLocked: z.boolean().optional(),
-  options: z.array(z.object({ value: z.string() })),
+  options: z.array(z.object({ value: z.string(), isGroup: z.boolean().optional() })),
```

Comment:
It was causing a groupOption to become regular option when created during the attribute creation itself.